pkgs/tools/virtualization/amazon-ec2-utils/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, curl
+, python3
+}:
+stdenv.mkDerivation rec {
+  pname = "amazon-ec2-utils";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "aws";
+    repo = "amazon-ec2-utils";
+    rev = version;
+    hash = "sha256:04dpxaaca144a74r6d93q4lp0d5l32v07rldj7v2v1c6s9nsf4mv";
+  };
+
+  buildInputs = [
+    python3
```

Comment:
You are sure this is not nativeBuildInputs?

Reply:
Yes, `ebsnvme-id` is a python script, so python is needed at runtime (and not at build time).